VMEncryption/main/oscrypto/centos_68/encryptstates/SplitRootPartitionState.py

```diff
@@ -216,7 +218,7 @@ def _append_boot_partition_uuid_to_fstab(self, boot_partition_uuid):
         contents += 'UUID={0}\t/boot\text2\tdefaults\t0 0'.format(boot_partition_uuid)
         contents += '\n'
 
-        with open('/etc/fstab', 'w') as f:
+        with open('/etc/fstab', 'wb') as f:
             f.write(contents)
```

Comment:
test cases - start as unicode, start as ascii, etc. -- make sure it preserves the intended format in all cases